.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
The gitignore file should contain the `test.md` file also. Kindly add that. :+1: